mods/default/dimensions.lua

```diff
@@ -32,6 +32,8 @@ mod:dimension {
 	biomes = {"default:grassland", "default:desert"},
 
 	sky = "default:sky_overworld",
+	
+	player_physics = { }
```

Comment:
This shouldn't be necessary

Reply:
Otherwise it prints the table error.

Reply:
Look my other comment related to that then, it's not supposed to print an error if you don't define this table
The error is only supposed to happen when a variable `player_physics` is defined + it's not a table

Reply:
My bad, I missed your other comment.